### DISCUSSION TITLE
ToC not showing

### DISCUSSION THREAD

**Discussion by imshubhamjain** [squidfunk/mkdocs-material#5843] [Q&A] [answered]
In my md file, ToC contain is not showing in right side.

I have already added the below extension in mkdocs.yml
markdown_extensions:
  - toc:
      permalink: true

**Comment by harryquit** [comment]
me too

**Comment by alexvoss** [comment]
Without a [minimal reproduction](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/) or at least some info there is no way we can provide advice. 

For the benefit of all who find this, tables of contents are produced out of the box, even without the extension being configured. There have to be some headings on the page for it to show. An empty table of contents will not be rendered.

**Comment by wangduanduan** [comment, marked as answer]
do not use top level `#` as toc prefix, top level `#` is used for title.

toc only for `##`, `###`, `####`, `#####`

# bad example
```
# ch1
abc

# ch2
abc
```

# good example
```
# some ch
## ch1
abc

## ch2
abc
```
